lib/logging/log.js

```diff
@@ -23,6 +23,9 @@ function NRlog (msg) {
         jsMsg = { ts: Date.now(), level: '', msg }
     }
     const date = new Date(jsMsg.ts)
+    if (typeof jsMsg.msg === 'object') {
```

Comment:
Would it be better to invert this? Thinking of other types that don't identify as object that ought to be stringified.

```suggestion
    if (typeof jsMsg.msg !== 'string') {
```

Reply:
Yeah, that makes sense